programs/us/us.cpp

```diff
@@ -771,9 +991,9 @@ void US_Win::help( int index )
       QMessageBox::information( this,
         tr( "About UltraScan..." ),
         tr( "UltraScan III version %1\n"
-            "%2\n"
-            "Copyright 1989 - 2021\n"
-            "Borries Demeler and the University of Texas System\n\n"
+            "( build %2 )\n"
```

Comment:
Could we add the branch or tag here? Just for the sake of reproducing the exact build